manifests/params.pp

```diff
@@ -138,10 +138,8 @@
 
   if downcase($facts['kernel']) == 'windows' {
     $zabbix_version = '4.4.5'
-  } elsif $facts['os']['name'] == 'Debian' and Integer($facts['os']['release']['major']) == 10 {
-    $zabbix_version =  '4.0'
```

Comment:
I dislike this because Debian 10 ships version 4.0 in the repo. So out of the box it would be wrong. Also using the external repository is not set by default. IMHO it should always prefer distro default versions, so that part is correct but the combination of the two isn't good.

Reply:
manage_repo is set be default to true. So it will always try to find the specified version from repo.zabbix.com.
I think changing it to 5.0  by default is okay for Debian as well.

Reply:
Hmm, personally I hate it when modules do that but since I don't use this module I guess I don't care too much.

Reply:
@ekohl manage_repo is true by default for Debian/Ubuntu: https://github.com/voxpupuli/puppet-zabbix/blob/master/manifests/params.pp#L16-L21
There were no zabbix packages for the old default version and we didn't want to bump that for every OS to avoid a backwards incompatible release. that's why we set it to zabbix 4 on debian 10 while implementing that.

Reply:
It was confusing because the default version on Debian 10 is also 4: https://packages.debian.org/buster/zabbix-agent (and other zabbix-* packages have matching versions). That's why I assumed no external repo was used.

Reply:
@ekohl can you approve this or is there anything left that should be changed?